exonum-light-client/README.md

```diff
@@ -1,8 +1,30 @@
 # Exonum Java Light Client
 Java client for [Exonum blockchain][exonum].
 
-## Quickstart
-If you are using Maven, add this to your pom.xml file
+## Overview
+Exonum light client is Java library for working with Exonum blockchain
+from the client side and can be easily integrated to existing 
+Java application.  
+Also, Exonum light client provides an access to [common utils][ejb-common]
+toolkit which contains some helpful functions for _hashing_,
+_serialization_ ect. 
+
+## Prerequisites
+- Java 8 or above is required for using this client
+- Maven 3.5 or above (only if you need to build it locally)
+
+## How to build
+To build the client locally, clone the repository, and
+run next command from the project's root 
+i.e. _exonum-java-binding_ directory:
+```bash
+mvn install -pl exonum-light-client -am
```

Comment:
If we do add build instructions here, shan't we make them reproducible (i.e., include the things that are described in the contribution guide, like `source tests_profile`)?